backend/server/router.go

```diff
@@ -0,0 +1,38 @@
+package server
+
+import (
+	"github.com/gorilla/mux"
+	"net/http"
+	"peoplemath/storage"
+	"time"
+)
+
+// Server struct to handle incoming HTTP requests
+type Server struct {
+	store        storage.StorageService
+	storeTimeout time.Duration
+}
+
+type Config struct {
```

Comment:
Why introduce a new `Config` type which has exactly the same members as `Server`, and then copy over those members one-by-one in `MakeHandler`? Why not just create the `Server` type directly?

My natural instinct here is that we should continue to instantiate the `Server` type as we are doing today (either via a struct literal or a static factory function) and keep `MakeHandler` as an instance method on `*Server`. Can you see any reason not to do that?

Reply:
My idea was to keep fields of Server struct not exported and make server configuring more explicit. But I think that you are right - we don't need it for now. I will refactor this.